src/Sylius/Behat/Context/Shop/CheckoutContext.php

```diff
@@ -0,0 +1,248 @@
+        $customer = $this->getService('sylius.factory.customer')->createNew();
+        $customer->setEmail($email);
+
+        $user->setCustomer($customer);
+        $user->setPlainPassword($password);
+        $user->addRole('ROLE_USER');
+        $user->addRole('ROLE_ADMIN');
+
+        $this->clipboard->setCurrentObject($user);
+
+        $this->persistObject($user);
+        $this->flushEntityManager();
+    }
+
+    /**
+     * @Given catalog has a product :productName priced at $:price
+     */
+    public function catalogHasAProductPricedAt($productName, $price)
+    {
+        /** @var ProductInterface $product */
+        $product = $this->getService('sylius.factory.product')->createNew();
+        $product->setName($productName);
+        $product->setPrice((int) $price);
+        $product->setDescription('Awesome star wars mug');
+
+        $channel = $this->clipboard->getCurrentObject('channel');
+        $product->addChannel($channel);
+
+        $this->persistObject($product);
+        $this->flushEntityManager();
+    }
+
+    /**
+     * @Given store allows paying offline
+     */
+    public function storeAllowsPayingOffline()
+    {
+        /** @var PaymentMethodInterface $paymentMethod */
+        $paymentMethod = $this->getService('sylius.factory.payment_method')->createNew();
+        $paymentMethod->setCode('PM1');
+        $paymentMethod->setGateway('offline');
+        $paymentMethod->setName('Offline');
+        $paymentMethod->setDescription('Offline payment method');
+        $paymentMethod->setFeeCalculatorConfiguration(array('amount' => 10));
+
+        /** @var ChannelInterface $channel */
+        $channel = $this->clipboard->getCurrentObject('channel');
+        $channel->addPaymentMethod($paymentMethod);
+
+        $this->persistObject($channel);
+        $this->persistObject($paymentMethod);
+        $this->flushEntityManager();
+    }
+
+    /**
+     * @Given I am logged in as :email
+     */
+    public function iAmLoggedInAs($email)
+    {
+        $this->getService('sylius.behat.security')->logIn($email, 'main', $this->getSession());
+    }
+
+    /**
+     * @Given I added product :name to cart
+     */
+    public function iAddedProductToCart($name)
+    {
+        /** @var ProductInterface $product */
+        $product = $this->getService('sylius.repository.product')->findOneBy(array('name' => $name));
+
+        $productShowPage = $this->getPage('Product\ProductShowPage')->openSpecificProductPage($product);
+        $productShowPage->pressButton('Add to cart');
+    }
+
+    /**
+     * @When I proceed selecting offline payment method
+     */
+    public function iProceedSelectingOfflinePaymentMethod()
+    {
+        $zone = $this->clipboard->getCurrentObject('zone');
+        $country = $this->clipboard->getCurrentObject('country');
+        $zoneMember = $this->clipboard->getCurrentObject('zone_member_country');
+
+        /** @var ShippingMethod $shippingMethod */
```

Comment:
I wonder if the fixture configured between this line and src/Sylius/Behat/Context/Shop/CheckoutContext.php#209 can be extracted to a dedicated method or even another class - fixture class?

Do you think its a good idea for reusing - mainly?